blog/02-02-2023-connect-and-interact-with-celo-using-web3onboard-library/connect-and-interact-with-celo-using-web3onboard-library.md

```diff
@@ -0,0 +1,1106 @@
+---
+title: Connect and interact with celo using web3Onboard library
+description: Alternative method of connecting to Celo networks
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [hardhat, celo, web3onboard]
+hide_table_of_contents: false
+slug: /tutorials/connect-and-interact-with-celo-using-web3onboard=library
+---
+
+![header](images/3.png)
+
+# Introduction
```

Comment:
Use ## instead of #

Reply:
Please read more on the guidelines for reviewing. The correct format is to use one '#' for the heading.